andela/ah-scorpion-frontend
# #159993333 Add SASS setup to enable use of .scss and .css files

#### What does this PR do?
- enables the use of SASS in the project

#### Description of tasks completed
- updated `package.json` file
  - added lines to add dependencies for `node-sass-chokidar` and `npm-run-all`
  - added scripts to run `SASS` build and watch when the project is started.

#### How can this PR be manually tested?
- clone the project by running `https://github.com/andela/ah-scorpion-frontend.git` using Terminal
- switch to branch `ch-SASS-setup-159993333`
- run `npm install` to install dependencies
- open the project in a code editor
- create a new `.scss` file with any name e.g `styles.scss`
- run `npm start` to start the server
- if you check your project directory, a new file `styles.css` was created on the same directory as the `styles.scss` (replace `styles` with the name of file you created)
- edit any lines in the `.scss` file
- the `.css` file will also be edited.
- delete the two file created above or ignore them using `.gitignore`

#### Background context
- Using SASS enables use of both `.scss` and `.css` files to style templates.
- css styles written in the `.scss` file are compiled to `.css` equivalent in a file with same name as the `.scss` file.

#### Related PT stories
- [#159993333](https://www.pivotaltracker.com/story/show/159993333)
